frontend/src/locales/scripts/en.json5

```diff
@@ -337,6 +337,21 @@
       title: "Search Settings",
       enable: "Enable Mature Content",
     },
+    safeBrowsing: {
+      title: "Safe Browsing",
+      desc: "Content marked as {sensitive} is not shown by default.",
+      sensitive: "sensitive",
+      toggles: {
+        fetchSensitive: {
+          title: "Sensitive results",
+          desc: "Show results marked as sensitive in the results area.",
+        },
+        blurSensitive: {
+          title: "Blurred content",
+          desc: "Images and texts are blurred to prevent seeing them without consent.",
```

Comment:
Whereas the previous toggle describes an action, this one just makes a statement. Should it be something like `Blur content` rather than `Blurred content`, and say something like "Blur images and text to prevent seeing sensitive material."?

Reply:
Agreed. I also don't think "Blurred" will necessarily translate well.

Reply:
Fixed in 543d2b9.